cumulus/parachains/runtimes/assets/asset-hub-rococo/tests/tests.rs

```diff
@@ -1672,4 +1674,69 @@ fn xcm_payment_api_works() {
 		Block,
 		WeightToFee,
 	>();
+
+	xcm_payment_api_foreign_asset_pool_works();
+}
+
+fn xcm_payment_api_foreign_asset_pool_works() {
```

Comment:
if you move this to `asset_test_utils::test_cases::` it would allow us to avoid copy/pasta the code for AH rococo, AH westend, AH Paseo, AH kusama and AH Polkadot

Reply:
Done. Moved the pool helper function as well